torch/nn/quantized/modules/linear.py

```diff
@@ -197,8 +191,9 @@ def __setstate__(self, state):
     def weight(self):
         return torch.ops.quantized.linear_unpack(self._packed_weight)
```

Comment:
maybe make unpack return both bias and weight then?

Reply:
It gets a little messy because bias is an optional tensor in this module and that is causing runtime error  `RuntimeError: sizes() called on undefined Tensor` when trying to call `torch::jit::script::Module::save`
We can perhaps add this in later if required? For now save bias in the module itself when it is set.

Reply:
Resolved this, bias was being set to undefined when there was no bias. I made it an optional Tensor as opposed to at::Tensor and it fixed the error.